src/ReinforcementLearningZoo/src/algorithms/policy_gradient/mpo.jl

```diff
@@ -224,7 +224,7 @@ end
 
 function solve_mpodual(Q::AbstractArray, ϵ)    
     g(η) = η * ϵ + η * mean(logsumexp( Q ./η .- Float32(log(size(Q, 2))), dims = 2))
-    Optim.minimizer(optimize(g, eps(ϵ), 10f0))
+    Optim.minimizer(optimize(g, eps(ϵ), maximum(abs.(Q))))
```

Comment:
I don't have any idea what this is doing, so I'm going to trust you blindly on this one. :)

Reply:
10 was an arbitrary upperbound I took from another implementation. But in practice I observed that it depends on the scale of the elements in Q. I guess the former implementer worked with smaller scales than I do.